server/src/Dao/RevisionDao.php

```diff
@@ -50,7 +51,7 @@ public function getUserRevisionsInCommonPages(
 	/**
 	 * Get common edited pages between multiple users
 	 *
-	 * @param int[] $users
+	 * @param array $users List of usernames/IP(s), ids, or mixed
```

Comment:
I don't think this supports `mixed`?
See the docblock on `getUserSearchColum()`